File: Linguagem de Programação I/Extras/Beecrowd_Extra/1448.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        scanner.nextLine(); 
        for (int i = 1; i <= t; i++) {
            String correct = scanner.nextLine();
            String team1 = scanner.nextLine();
            String team2 = scanner.nextLine();

            int score1 = 0;
            int score2 = 0;
            for (int j = 0; j < correct.length(); j++) {
                if (correct.charAt(j) == team1.charAt(j)) {
                    score1++;
                }
                if (correct.charAt(j) == team2.charAt(j)) {
                    score2++;
                }
            }

            System.out.println("Instancia " + i);
            if (score1 > score2) {
                System.out.println("time 1");
            } else if (score1 < score2) {
                System.out.println("time 2");
            } else {
                boolean tieBreaker = false;
                for (int j = 0; j < correct.length(); j++) {
                    if (correct.charAt(j) == team1.charAt(j) && correct.charAt(j) != team2.charAt(j)) {
                        System.out.println("time 1");
                        tieBreaker = true;
                        break;
                    } else if (correct.charAt(j) != team1.charAt(j) && correct.charAt(j) == team2.charAt(j)) {
                        System.out.println("time 2");
                        tieBreaker = true;
                        break;
                    }
                }
                if (!tieBreaker) {
                    System.out.println("empate");
                }
            }
            System.out.println();
        }
        scanner.close();
    }
}

File: Linguagem de Programação I/Extras/Beecrowd_Extra/1448.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	scanner.Scan()
	t, _ := strconv.Atoi(scanner.Text())
	for i := 1; i <= t; i++ {
		scanner.Scan()
		correct := scanner.Text()

		scanner.Scan()
		team1 := scanner.Text()

		scanner.Scan()
		team2 := scanner.Text()

		score1 := 0
		score2 := 0
		for j := 0; j < len(correct); j++ {
			if correct[j] == team1[j] {
				score1++
			}
			if correct[j] == team2[j] {
				score2++
			}
		}

		fmt.Printf("Instancia %d\n", i)
		if score1 > score2 {
			fmt.Println("time 1")
		} else if score1 < score2 {
			fmt.Println("time 2")
		} else {
			tieBreaker := false
			for j := 0; j < len(correct); j++ {
				if correct[j] == team1[j] && correct[j] != team2[j] {
					fmt.Println("time 1")
					tieBreaker = true
					break
				} else if correct[j] != team1[j] && correct[j] == team2[j] {
					fmt.Println("time 2")
					tieBreaker = true
					break
				}
			}
			if !tieBreaker {
				fmt.Println("empate")
			}
		}
		fmt.Println()
	}
}